backend/backend/api/routes.py

```diff
@@ -346,4 +353,18 @@ async def github_comment(repo, pr_number, request):
         with gr.Column():
             github_comment_output = gr.Textbox(label="Response")
     github_comment_button.click(github_comment, inputs=[
-                                repo_input, pr_number_input, request_input], outputs=github_comment_output)
+                                repo_input, pr_number_input, request_input],
+                                outputs=github_comment_output)
```

Comment:
The github_comment_button.click() function is called with different inputs for the same button. This could lead to inconsistent results.